product-scenarios/1-manage-users-and-roles/1.1-admin-user-manage-roles/1.1.2-manage-roles-with-scim1/src/test/java/org/wso2/identity/scenarios/test/scim/ManageSecondaryUserStoreSCIM1TestCase.java

```diff
@@ -0,0 +1,169 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.identity.scenarios.test.scim;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.json.simple.JSONObject;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.test.utils.dbutils.H2DataBaseManager;
+import org.wso2.carbon.identity.user.store.configuration.stub.dto.PropertyDTO;
+import org.wso2.carbon.identity.user.store.configuration.stub.dto.UserStoreDTO;
+import org.wso2.identity.scenarios.commons.SCIM1CommonClient;
+import org.wso2.identity.scenarios.commons.ScenarioTestBase;
+import org.wso2.identity.scenarios.commons.clients.service.client.UserStoreConfigAdminServiceClient;
+import org.wso2.identity.scenarios.commons.util.UserStoreConfigUtils;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.wso2.identity.scenarios.commons.util.Constants.IS_HTTPS_URL;
+import static org.wso2.identity.scenarios.commons.util.IdentityScenarioUtil.getJSONFromResponse;
+
+public class ManageSecondaryUserStoreSCIM1TestCase extends ScenarioTestBase {
+
+    private static final String DOMAIN_ID = "USERSTOREDB";
+    private static final String USER_STORE_DB_NAME = "JDBC_USER_STORE_DB";
+    private static final String DB_USER_NAME = "wso2carbon";
+    private static final String DB_USER_PASSWORD = "wso2carbon";
+    private UserStoreConfigAdminServiceClient userStoreConfigAdminServiceClient;
```

Comment:
Better to add a space between constant and variable declaration.

Reply:
added